app/v1/connectors/DesBaseConnector.scala

```diff
@@ -45,8 +45,8 @@ trait DesBaseConnector {
           valid => Right(valid)
         )
       case status =>
-        logger.warn(s"[$parserName][read]: Unexpected response, status $status returned")
-        Left(UnexpectedFailure(response.status,s"Status ${response.status} $unexpectedErrorMessage"))
+        logger.warn(s"[$parserName][read]: Unexpected response, status $status returned with body ${response.body}")
+        Left(UnexpectedFailure(response.status,response.body))
```

Comment:
Happy to approve for testing but do we want to expose errors from des to users of our API? I'd argue maybe it's better to be returned just in the logs, but then I'm also not sure what kind of messages we might get from downstream

Reply:
whooops i was meant to revert that! i jsut want the warn....